examples/aspnet-webapi-zip/README.md

```diff
@@ -0,0 +1,44 @@
+# ASP.NET ZIP
+
+This example shows how to deploy an ASP.NET application using the Lambda Web Adapter packaged as a ZIP file.
+
+The `dotnet6` runtime is used.
+
+```yaml
+Resources:
+  AspNetOnLambdaWebAdapterFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: src/
+      Handler: run.sh
```

Comment:
One tip here: since your app is a single binary to startup. You can skip `run.sh`, and put the binary name here, e.g. 

```yaml
      Handler: AspNetWebApi
```